packages/rum/webpack.config.js

```diff
@@ -41,76 +40,29 @@ const devConfig = entry => ({
     library: '[name]',
     libraryTarget: 'umd'
   },
-  mode: 'development',
-  stats: {
-    assets: true,
-    modules: false
-  },
-  node: false,
-  module: {
-    rules: [
-      {
-        test: /\.js$/,
-        loader: 'babel-loader',
-        options: getBabelConfig(BUNDLE_TYPES.BROWSER_DEV)
-      }
-    ]
-  },
-  plugins: [
-    new EnvironmentPlugin({
-      NODE_ENV: 'development'
-    })
-  ]
+  ...getWebpackReleaseConfig(BUNDLE_TYPES.BROWSER_DEV)
```

Comment:
I think it's better to put this at the top, so it will be overwritten by our custom config in this file! 
I can see there are no conflicting configs at the moment, but to make sure we get the order right.